### PR TITLE
Fix Sec Stairs & Run MapMerge

### PR DESCRIPTION
**Pull request by KillianKirilenko**
Excuse the extremely large diffs, but I've taken this opportunity to run MapMerge on the six main levels whilst fixing the broken stairs in Security.

**HOW TO MAPMERGE, THE HARD WAY, AFTER THIS PR**
Make sure you're using the version in tools/mapmerge2.

Step by step:
- Before opening your map editor (e.g. DM), run Prepare Maps.bat
- You may now open your map editor and edit the map
- Save your changes and **close your map** (Ctrl+W or the little x on the tab)
- Run Merge Maps.bat
- When prompted, select levels you want merged. You may want to scroll up to see the full list. You can indicate a range with hyphen (e.g. 1-4) or specific maps with commas (e.g. 1,2,4).
- Merge it!
- Commit!

I have no idea how to set up the hooks so you're on your own with that one.

### CODE REVIEW

**Comment by RadiantFlash7** [comment]
@killian Mind doing me a quick favor, and fixing the missing cable leading to tech storage next to engineering?

**Comment by KillianKirilenko** [comment]
One sec.

**Comment by TheFurryFeline** [comment]
Wish the MapDiffBot was working right. T_T

File: .gitignore [hunk 1]
```diff
@@ -35,3 +35,9 @@ temp.dmi
 /maps/RandomZLevels/backup
 /tools/midi2piano.zip
 /maps/yw/backup
+maps/yw/cryogaia-04-maintenance.dmm.before
+maps/yw/cryogaia-05-main.dmm.before
+maps/yw/cryogaia-03-transit.dmm.before
+maps/yw/cryogaia-02-mining.dmm.before
+maps/yw/cryogaia-01-centcomm.dmm.before
+maps/yw/residential/residential.dmm.before
```
Comment (TheFurryFeline) [inline .gitignore:43]: You can just delete these from your branch's files with File Explorer.

**Pull request merged**